src/pages/en/guides/markdown-content.md

```diff
@@ -459,6 +459,28 @@ export default {
 }
 ```
 
+#### Markdown heading IDs
+
+Astro injects an `id` attribute into all heading elements (`<h1>` to `<h6>`) in Markdown and MDX files and provides a `getHeadings()` utility for retrieving these IDs in [Markdown exported properties](#exported-properties).
+
+You can customize heading IDs by adding an alternative plugin that injects `id` attributes (e.g. `rehype-slug`). Your custom IDs will be reflected in HTML output and the items returned by `getHeadings()`.
```

Comment:
```suggestion
You can customize these heading IDs by adding a rehype plugin that injects `id` attributes (e.g. `rehype-slug`). Your custom IDs, instead of Astro's defaults, will be reflected in HTML output and the items returned by `getHeadings()`.
```
"an alternate" plugin maybe sounds like there is one, official sanctioned alternative? Are they all rehype plugins that do this? If so, what about saying that?

Reply:
Right it would be a rehype plugin. `rehype-slug` is the most common (? citation needed) but there are others I think. Or you could write your own if you were feeling adventurous.